deploy/runtime/versions.lk

```diff
@@ -51,10 +51,38 @@ Some more examples:
 //***********************************************************************************************************************************************************
 //CURRENT WORKING VERSION (see SAM/main_add.h) - CHANGE THIS VERSION NUMBER WHEN RELEASES ARE CREATED AND MAKE A NEW CURRENT WORKING VERSION FUNCTION
 
-version_upgrade{'2023.9.19'} = define() {
+version_upgrade{'2023.10.31'} = define() {
 
 	const cfg = config();
 
+	// SAM 2022.11.21 default inverters are not in new CEC inverters library
+	if ( cfg.tech == 'Flat Plate PV' || cfg.tech == 'PV Battery' || cfg.tech == 'High-X Concentrating PV' ) 
+	{
+		old_inverter = value( 'inv_snl_inverter_name' );
```

Comment:
When I run the upgrade script old_inverter returns as null: 

![image](https://github.com/NREL/SAM/assets/54449384/251adaca-96a5-4157-bea5-29f492f9aca7)

Then this notice pops up after choosing a new inverter from the list: 

![image](https://github.com/NREL/SAM/assets/54449384/e6c2cef7-d7a8-441c-8ddf-5efb6f1ac1d7)

Maybe that's normal but wanted to note.

Reply:
![image](https://github.com/NREL/SAM/assets/54449384/2cd01599-5ce0-4db7-ba8c-b0bac14037cd)